src/Table/read_property_tree_as_votable/read_resource/read_table/read_data/read_tabledata/read_tabledata.cxx

```diff
@@ -81,11 +81,17 @@ void tablator::Table::read_tabledata(const boost::property_tree::ptree &tabledat
                     insert_ascii_in_row(
                             columns[column].type, columns[column].array_size, column,
                             element, offsets[column], offsets[column + 1], row_string);
+                } catch (std::runtime_error &error) {
+                    throw std::runtime_error("Failed to write value from row " +
+                                             std::to_string(current_row + 1) +
+                                             ", field " + std::to_string(column) +
+                                             ". Error message: " + error.what());
                 } catch (std::exception &error) {
                     throw std::runtime_error(
-                            "Invalid " + to_string(fields[column].type) + " in row " +
-                            std::to_string(current_row + 1) + ", field " +
-                            std::to_string(column) + ".  Found '" + element + "'");
+                            "Invalid " + to_string(fields[column].type) + "value " +
+                            element + " in row " + std::to_string(current_row + 1) +
+                            ", field " + std::to_string(column) +
+                            ". Error message: " + error.what());
```

Comment:
Did you mean "Failed to read value" instead of "Failed to write value"? Also, the responses to these 2 exception types are nearly identical. Can they be consolidated into a single catch?